Topics/Tech_Stacks.md

```diff
@@ -1 +1,28 @@
 ## Tech Stacks
+1. React Native
+    1. Set up
+        1. Installation
+            1. Install Node.js and npm (Node Package Manager) on your computer. You can download and install Node.js from the official website: https://nodejs.org/en/
+                #Note: Suggested to download the LTS version because it is generally more stable than the Current or Latest version.
+            2. Install expo go on your ios device (https://apps.apple.com/us/app/expo-go/id982107779) or android device (https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1)
+            3. Download VS Code Extensions
+                1. ES7+ React/Redux/React-Native snippets (Extension ID: dsznajder.es7-react-js-snippets)
+                2. React Native Tools (Extension ID: msjsdiag.vscode-react-native)
```

Comment:
Curious, what are these extensions for? Are there similar extensions for users who do not use VS Code?